prov/rxm/src/rxm_conn.c

```diff
@@ -511,7 +511,7 @@ void rxm_process_connect(struct rxm_eq_cm_entry *cm_entry)
 		rxm_set_peer_flow_ctrl(conn, cm_entry->data.accept.flow_ctrl);
 	}
 
-	if (conn->flow_ctrl & conn->peer_flow_ctrl) {
```

Comment:
Can we change the definitions for flow_ctrl and peer_flow_ctrl to bool, and update all places they are accessed accordingly?

Reply:
Yes, will make the change. This will make the codes more readable and consistent with the flow_ctrl ops (available) return type.